spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Location.java

```diff
@@ -270,14 +291,13 @@ static LogicalLocation fromStackTraceElement(@NonNull StackTraceElement element)
         static Optional<LogicalLocation> fromBugInstance(@NonNull BugInstance bugInstance) {
             Objects.requireNonNull(bugInstance);
             ClassAnnotation primaryClass = bugInstance.getPrimaryClass();
-            SourceLineAnnotation sourceLine = bugInstance.getPrimarySourceLineAnnotation();
             return bugInstance.getAnnotations().stream().map(annotation -> {
                 String kind = findKind(annotation);
                 if (kind == null) {
                     return null;
                 }
                 String name = annotation.format("givenClass", primaryClass);
-                return new LogicalLocation(name, null, kind, sourceLine.format("full", primaryClass), null);
+                return new LogicalLocation(name, null, kind, null, null);
```

Comment:
for now, i'm removing the fullyQualified property for one reason: that property should point to the method/parameter/etc. Right now, it is almost the same as the physical location, which is wrong.

When I was taking a look, I saw that the annotations have the content that we need, but I couldn't figure out yet how annotation to choose to add here. Because, some times, we have a few: